msteams-platform/concepts/device-capabilities/mobile-camera-image-permissions.md

```diff
@@ -1,34 +1,31 @@
 ---
-title: Camera and image capabilities in Teams
-description: How to use Teams Javascript client SDK to enable native camera and image capabilities
-keywords: camera image capabilities native device permissions
+title: Integrate media capabilities 
+description: How to use Teams Javascript client SDK to enable media capabilities
+keywords: camera image microphone capabilities native device permissions media
 ms.topic: conceptual
 ms.author: lajanuar
 ---
 
-# Camera and image capabilities in Teams
+# Integrate media capabilities 
 
->[!IMPORTANT]
->
-> * At present, Teams support for camera and image capabilities is only available for mobile clients.
->* The `selectMedia`, `getMedia`, and `viewImages` APIs can be invoked from multiple Teams surfaces such as task modules, tabs, and personal apps. For more details, _see_ [Entry points for Teams apps](../extensibility-points.md)
+This document gives a walkthrough on integrating  native device capabilities with Teams platform.
 
-You can use the  [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), to easily integrate camera and image capabilities within your Microsoft Teams mobile app. The SDK provides the tools necessary for your app to access a user’s [device permissions](native-device-permissions.md?tabs=desktop#device-permissions) and build a richer experience.
+The process with which you combine the native device capabilities, such as **camera**, and **microphone** along with Teams platform is called integration of media capabilities. 
```

Comment:
No need to explain what integration means as it's well understood in dev community.
```suggestion
```